client/my-sites/patterns/components/pattern-preview/index.tsx

```diff
@@ -229,6 +229,55 @@ function PatternPreviewFragment( {
 		} );
 	};
 
+	const getPatternPreviewHeader = () => (
```

Comment:
```suggestion
	const patternPreviewHeader = (
```

Is there any reason why we would want to declare this as a function? I'd suggest declaring this as a simple constant, since we're not accepting any parameters for conditional rendering.

Reply:
> Is there any reason why we would want to declare this as a function?

Not really. In hindsight, a constant is better. 

However, this is no longer needed due to adoption of the CSS-only solution.